app/controllers/admin/admins_controller.rb

```diff
@@ -1,5 +1,8 @@
 class Admin::AdminsController < ApplicationController
   before_action :logged_in_user, :is_admin?
 
-  def index; end
+  def index
+    @users = User.admin.page(params[:page])
+                 .per(Settings.per_page.per_10).order_by_name
```

Comment:
nên gom scope data trước khi chia page
```suggestion
    @users = User.admin.order_by_name.page(params[:page])
                 .per(Settings.per_page.per_10)
```

Reply:
Dạ chị. Em fix mấy kia luôn. Em cám ơn ạ.